src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,14 @@
-export const TodoInfo = () => {};
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo:React.FC<{ todo: Todo }> = ({ todo }) => {
+  const userInfo = todo.user && <UserInfo user={todo.user} />;
```

Comment:
Actually, you can place `todo.user && <UserInfo user={todo.user} />` directly inside jsx